django/contrib/sessions/backends/base.py

```diff
@@ -18,6 +19,13 @@
 # on case insensitive file systems.
 VALID_KEY_CHARS = string.ascii_lowercase + string.digits
 
+# the delimiter cannot be a member of VALID_KEY_CHARS or a file directory
+# component. It can only be a single character.
+SESSION_KEY_DELIMITER = '$'
+
+# this should be available in hashlib
+SESSION_HASHING_ALGORITHM = 'sha256'
+
```

Comment:
`SESSION_HASHED_KEY_PREFIX = SESSION_HASHING_ALGORITHM + SESSION_KEY_DELIMITER`

You'll see why later.